tests/tests.toml

```diff
@@ -234,6 +234,11 @@ tokens = true
 filename = "tokens/indent1.py"
 tokens = true
 
+[[test]]
+filename = "../integration_tests/test_max_min.py"
+asr = true
+ast = true
```

Comment:
We don't need to test AST, unless there was a specific problem with the parser in this PR. The parser pretty much works for any Python code already, so we don't need to test it. Every time we add a test, it slows down the test suite, so we should only test "new features", in this cast just ASR.